exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -205,4 +211,35 @@ public JaegerGrpcSpanExporter build() {
       return new JaegerGrpcSpanExporter(serviceName, channel, deadline);
     }
   }
+
+  /**
+   * Configure and install Jaeger exporter with default settings. Configuration can be provided via
+   * system properties and environmental variables: {@code JAEGER_ENDPOINT} e.g. {@code
+   * localhost:14250} and {@code JAEGER_SERVICE_NAME} e.g. {@code my-deployment}.
+   *
+   * @param tracerProvider tracer provider
+   */
+  public static void installDefault(TracerSdkProvider tracerProvider) {
```

Comment:
Support ENV vars has been asked here https://github.com/open-telemetry/opentelemetry-java/issues/797#issuecomment-590439077.

Given the main functionality is supporting env vars the name of the method should reflect that. Also I do not like the static method as there is a builder. So I have moved the code into the builder, basically the same design is in core Jaeger client, where builders can be constructed from env properties